src/controllers/redeploy.go

```diff
@@ -0,0 +1,8 @@
+package controllers
+
+func redeploy(callbackID string, data map[string]interface{}) {
+	err := stop(callbackID, data)
+	if err == nil {
+		deploy(callbackID, data)
```

Comment:
This will not work, as there are no subdomain, server and access fields in the data map which the deploy function relies on.

Reply:
Fixed.